tests/pytests/test_newhorizons_drivers.py

```diff
@@ -35,14 +35,40 @@ def test_kernels(scope="module"):
         for kern in kern_list:
             os.remove(kern)
 
+# Test load of newhorizons labels
+@pytest.mark.parametrize("image", ["mc3_0295574631_0x536_sci"])
+def test_nhmvic_load(test_kernels, image):
+    label_file = get_image_label(image, 'isis')
+    isd_str = ale.loads(label_file, props={'kernels': test_kernels[image],
+                                           'exact_ck_times': False})
+    compare_isd = image_dict[image]
+
+    isd_obj = json.loads(isd_str)
+    assert compare_dicts(isd_obj, compare_isd) == []
+
+
+class test_mvic_isis3_naif(unittest.TestCase):
+
+    def setUp(self):
+        label = get_image_label("mc3_0295574631_0x536_sci", "isis")
+        self.driver = NewHorizonsMvicIsisLabelNaifSpiceDriver(label)
+
+    def test_instrument_id(self):
+        assert self.driver.instrument_id == 'NH_MVIC'
+
+    def test_ikid(self):
+        assert self.driver.ikid == -98908
+
+    def test_sensor_model_version(self):
+        assert self.driver.sensor_model_version == 1
+
```

Comment:
I think are some properties in the driver that are untested under the test_mvic_isis3_naif unittest suite, like the `ephemeris_start_time` and `ephemeris_stop_time` to name some